hist/hist/src/TGraph.cxx

```diff
@@ -2456,35 +2457,35 @@ Double_t **TGraph::ShrinkAndCopy(Int_t size, Int_t oend)
 /// ~~~
 
 void TGraph::Sort(Bool_t (*greaterfunc)(const TGraph*, Int_t, Int_t) /*=TGraph::CompareX()*/,
-                  Bool_t ascending /*=kTRUE*/, Int_t low /* =0 */, Int_t high /* =-1111 */)
+                  Bool_t ascending /*=kTRUE*/, Int_t low /*=0*/, Int_t high /*=-1111*/)
 {
-
-   // set the bit in case of an ascending =sort in X
-   if (greaterfunc == TGraph::CompareX && ascending  && low == 0 && high == -1111)
+   // set the bit in case of an ascending =sort in X   
+   if (greaterfunc == TGraph::CompareX && ascending && low == 0 && high == -1111)
```

Comment:
I don't know why github highlights these two lines as changed, there doesn't seem to me anything changed here. Can you double check?